modules/Hoot/ui/modals/exportData.js

```diff
@@ -39,7 +39,7 @@ export default class ExportData {
         this.appendToFgdbCheckbox = this.container.select( '#exportAppendFgdb' );
         this.dataExportNameTextInput = this.container.select( '#dataExportNameTextInput' );
         this.submitButton = this.container.select( '#exportDatasetBtn' );
-        this.submitButton.attr('disabled', false);
+        this.submitButton.attr('disabled', null);
```

Comment:
This clears the attribute, instead of changing it's value.  Makes export data dialog button enabled with defaults.